include/effects.h

```diff
@@ -944,8 +953,8 @@ typedef struct RadialShimmerFXData {
     /* 0x1C */ f32 unk_1C;
     /* 0x20 */ f32 unk_20;
     /* 0x24 */ s32 unk_24;
-    /* 0x28 */ s32 unk_28;
-    /* 0x2C */ s32 unk_2C;
+    /* 0x28 */ s32 timeLeft;
+    /* 0x2C */ s32 lifeTime;
```

Comment:
Can you define the difference? Is lifeTime the max

Reply:
timeLeft counts down, lifeTime counts up. most effects have these, and we decided to name them this way a while ago. we can change them at some point though if we don't like the names